app/src/test/java/pm/gnosis/heimdall/ui/safe/SafeOverviewViewModelTest.kt

```diff
@@ -204,5 +223,58 @@ class SafeOverviewViewModelTest {
         testObserver.assertError(exception)
     }
 
+    @Test
+    fun shouldShowLowBalanceViewOnHighBalance() = onTestableComputationScheduler {
```

Comment:
Can we test the following cases:
* Repeat request without error and with error
* Once dismissed it should not show the notification
* Dismissed should change back once a higher balance is returned